index.bs

```diff
@@ -1874,15 +1883,63 @@ This document mostly covers API design for the Web,
 but those who design APIs are hopefully also writing specifications
 for the APIs that they design.
 
-<h3 id="writing-resources">Other resources</h3>
-
-Some useful advice on how to write specifications is available elsewhere:
-  * <a href="https://ln.hixie.ch/?start=1140242962&amp;count=1">Writing
-    specifications: Kinds of statements</a> (Ian Hickson, 2006)
-  * <a href="https://www.w3.org/TR/qaframe-spec/">QA Framework:
-    Specification Guidelines</a> (W3C QA Working Group, 2005)
+<h3 id="requirements-on-authors-and-implementers">Identify the audience of each requirement in your specification</h3>
+<!-- was "Conformance for Documents and Implementations" in the HTML Design Principles -->
+
+The web platform is,
+unusually,
+designed to be robust in accepting poorly formed markup.
+
+When writing specifications for the web,
+be sure to note authoring requirements
+for well formed markup (the "conforming language"),
+and also
+implementation requirements
+for handling poorly formed markup (the "supported language").
+
+This allows us to
+have a relatively clean and understandable language for authors,
+while at the same time supporting existing documents
+that make use of older or nonstandard constructs,
+and enabling better interoperability in error handling.
+
+Some of the design principles in this document apply much more
+to conformance requirements for content (the "conforming language")
+while others apply much more
+to conformance requirements for implementations (the "supported language").
+Since the supported language is a strict superset of the conforming language,
+there is considerable overlap,
+but we have tried to be clear
+which set of requirements each principle applies to.
+
+<h3 id="avoid-ambiguity">Specify completely and avoid ambiguity</h3>
+<!-- was "Well-defined behavior" in the HTML Design Principles -->
+
+If the specification leaves room for implementations to make different choices
+which mean authors need to write different markup for different implementations,
+this adds a maintenance burden for authors. 
+
+The specification should be complete enough 
+for implementations to be interoperable,
+without depending on or referring to the details of other implementations.
+and there isn't a good reason to allow variation between implementations,
```

Comment:
This has gone a bit funny here. Can we delete the remainder of this text, or most of it?

Reply:
Fixed, I think.